test/src/test/java/hudson/security/TokenBasedRememberMeServices2Test.java

```diff
@@ -128,7 +129,7 @@ public void basicFlow() throws Exception {
         wc.executeOnServer(() -> {
             Authentication a = Jenkins.getAuthentication2();
             assertEquals("bob", a.getName());
-            assertEquals(ImmutableList.of("authenticated", "myteam"), a.getAuthorities().stream().map(GrantedAuthority::getAuthority).collect(Collectors.toList()));
+            assertEquals(Arrays.asList("authenticated", "myteam"), a.getAuthorities().stream().map( GrantedAuthority::getAuthority).collect( Collectors.toList()));
```

Comment:
[nit] Here and elsewhere, there seem to be some unrelated whitespace changes to code that isn't being changed by this PR. It would be ideal to revert such changes to eliminate noise and keep this PR focused on the important changes.